src/app/shared/components/template/components/combo-box/combo-box.component.html

```diff
@@ -3,22 +3,24 @@
     class="open-combobox margin-t-regular"
     (click)="openModal()"
     [class.placeholder-style]="(!_row.value && placeholder) || prioritisePlaceholder"
+    [class.disabled]="disabled()"
```

Comment:
nit(non-blocking): we currently have 3 separate class bindings alongside an ngClass binding. Might be tidier to use a single computed property that defines all classes from within the `ts` file and just passes into html

Reply:
I tried using a single computed property, but since some of the classes depend on the non-signal `_row.value`, the values don't update properly. For now I've just consolidated them into a single `ngClass` binding, see https://github.com/IDEMSInternational/open-app-builder/pull/2733/commits/6f594741734d421ac9931bf0203628be537d8e3f

Reply:
There is a separate `rowSignal()` signal that can be used generally instead of `_row`, and then a more fine-grained `value()` signal that can be used instead of `_row.value` (see full list on `src\app\shared\components\template\components\base.ts`), but in either case not a huge issue

Reply:
I did try implementing this, but I found that the `value()` signal didn't seem to update as expected, nor trigger my computed property that referenced it to update. Logging out the value of `rowSignal().value` and `value()` suggested that the former was updating with updating the latter. After spending more time than I meant to trying to resolve it, I opted to keep the logic in the template for now (and delete some WIP code I accidentally committed to this PR), see #2744